content/brazilian-portuguese/jsappybird/_index.md

```diff
@@ -0,0 +1,58 @@
+---
+title: "JS: JSappyBird"
+description: "Crie Seu Próprio Jogo JSappyBird"
+date: 2023-09-11T13:24:17-07:00
+prereq: "nenhum"
+difficulty: "Iniciante"
+icon: "fab fa-html5"
+draft: false
+---
+
+## Introdução
+
+Você já ouviu falar do Flappy Bird, não é mesmo? Neste curso, vamos criar juntos a nossa versão desse joguinho usando algo chamado JavaScript! E aí, preparado para essa aventura divertida?
+
+<p style="text-align: center;"><iframe src="https://giphy.com/embed/euuaA2cwLEUuI" width="350" height="350" frameBorder="0" class="giphy-embed"></iframe>
+
+Com certeza você já ouviu falar do jogo Flappy Bird, né? Neste curso, vou te ensinar a fazer o seu próprio joguinho! E o melhor: você poderá chamar seus amigos para jogar junto! Juntos, vamos aprender a usar a linguagem de programação chamada JavaScript e fazer o nosso passarinho pular e voar pelo mundo que você vai criar. Vai ser super divertido!.
+
+## Começando
+
+O objetivo de hoje é aprender algumas coisinhas em JavaScript e usar o que aprendemos para criar um jogo estilo Flappy Bird! Vamos preparar o jogo.
+
+Hoje, vamos usar o Replit. Antes de começar, você precisa criar uma conta no Replit, caso ainda não tenha uma. 
+
+Clique no botão abaixo para se registrar no Replit - você precisará de um e-mail para isso. Se você já tem uma conta no Replit, também pode usá-la clicando o botão abaixo para entrar.
+
+<a class="my-2 mx-4 btn btn-info" href="https://replit.com/signup" target="_blank">Criar minha conta no Replit</a>
+
+<img src="./img/replit-signup.png" height="300" alt="Página do Replit para criar uma conta" />
+
+Em seguida, abra o console do Replit que usaremos hoje clicando no botão abaixo. Você verá a tela que está aí embaixo.
+
+<a class="my-2 mx-4 btn btn-info" href="https://replit.com/@nuevofoundation/JSappyBird-CSWeek" target="_blank">Rode o Replit</a>
+
+<img src="./img/replit-start-screen.png" height="300" alt="Página inicial do Repl" />
+
+Agora, vamos fazer uma cópia desse jogo para que possamos fazer nossas próprias mudanças. Clique no botão "Fork Repl" no canto superior direito e depois no botão "Fork Repl" novamente.
+
+<img src="./img/replit-fork.png" height="300" alt="Copiando o projeto no programa replit" />
+
+Depois de copiar o Replit, você verá uma janela parecida com a que está logo abaixo:
+
+<img src="./img/replit-window.png" height="300" alt="Janela do Replit " />
+
+We will ignore some of the details on this screen, but we will go over some of the basics that show up. The `index.html` file, as well as all the files ending with `.js` contain code, or instructions for the computer to run. By pressing the **run** button, you tell the computer to perform the instructions in the code. Any results from running the code will be displayed on the right side, called the **console**.
```

Comment:
Please translate

Reply:
My mistake, it was translated under this sentence, I forgot to remove the English's block that I use as reference :)